src/hooks/useActionCenterSubscribers/useThorchainLpWithdrawActionSubscriber.tsx

```diff
@@ -90,21 +88,19 @@ export const useThorchainLpWithdrawActionSubscriber = () => {
         return
       if (action.type !== ActionType.Withdraw) return
 
-      const { thorMemo, txHash } = action.transactionMetadata
+      const { txHash } = action.transactionMetadata
 
       // Check if the transaction is confirmed on the blockchain
       const accountId = action.transactionMetadata.accountId
       if (!accountId) return
 
-      const accountAddress = fromAccountId(accountId).account
-      const serializedTxIndex = serializeTxIndex(
-        accountId,
+      // Note: looking by serializedTxIndex won't necessarily work, as for out Txs, the memo and origin memo may be different
+      // we *do* have logic to get extra metadata, including the originMemo, however, it is not guaranteed to be here by the time we hit this
+      // so for the sake of simplicity, we simply do a lookup by txHash, which does the do
```

Comment:
kept `originMemo` here to keep intent explicit but yeah, lost a day with this,  and in the end, Txid is all we need. 
Using `originMemo` here was just to keep things aligned with the original lookup by Tx index but tbf.. why? It's just extra data we don't need anyway, a Tx hash is a Tx hash

*did* keep the originMemo logic in chain-adapters (and the filter selector, although unused anywhere, happy to remove that one) as I feel like that's extra metadata we absolutely want to leverage *if* and *when* available, for the sake of completeness. If we really feel strongly against it, can remove that too!